AutomatedTesting/Gem/PythonTests/editor/test_Editor_Periodic_Optimized.py

```diff
@@ -0,0 +1,61 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import os
+import pytest
+
+import ly_test_tools.environment.file_system as file_system
+from ly_test_tools.o3de.editor_test import EditorSingleTest, EditorSharedTest, EditorParallelTest, EditorTestSuite
+
+
+@pytest.mark.SUITE_periodic
```

Comment:
why not move the period tests along with the main one? It should run in less than the 3 minutes :)

Reply:
Good point. Moved!